api/scpca_portal/models/computed_file.py

```diff
@@ -447,3 +459,29 @@ def delete_s3_file(self, force=False):
             return False
 
         return True
+
+    def generate_content_descriptions(self):
```

Comment:
Let's get rid of this for now, since the required properties will be on the computed file.

Reply:
Yeah, I should've removed it -- it's unused.